src/axom/mint/docs/sphinx/sections/cell_types.rst

```diff
@@ -0,0 +1,37 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+
+.. |quad| ..image:: ../figures/quad.png
+          :scale: 100%
+          :alt: Linear Lagrangian Quadrilateral Element
```

Comment:
For some reason, I couldn't get those replacements to work properly ((used in fem.rst)  inlining those images doesn't do the right thing either. This used to work with prior to integrating the Mint user guide with unified Axom Sphinx documentation. I'll try to see if I can resolve this somehow